### ISSUE TITLE
If microsimulation worker crashes, raise some sort of error to allow API (and subsequently, app) to show error message

### ISSUE THREAD

**Issue by anth-volk** [PolicyEngine/policyengine-api#1815]
The API never actually fails if the worker does when calculating microsimulation outputs. It'd be great to have some sort of error raising behavior that allows the app to show an error message.

**Comment by nikhilwoodruff** [comment]
I think this is fixed now? e.g. you can see error stack traces for the UK simulations that fail.

**Comment by anth-volk** [comment]
True, but what happens is that the API itself will continue to emit GET requests to the endpoint endlessly, causing a spinner, unless I'm mistaken.

**Comment by nikhilwoodruff** [comment]
Do you mean the `economy` endpoint? Or are we talking about household impacts

**Comment by anth-volk** [comment]
`economy` endpoint
